end_to_end/gemma/7b/2_test_gemma.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# This file is both an integration test that runs once a day on a v4-16 and documentation for how to get started with Gemma-7b. 
+# Please make sure you have run end_to_end/gemma/7b/1_test_gemma.sh before running commands from this file. 
+
+# The flow of this file is as follows:
+# 1. Convert the checkpoint downloaded from Kaggle to make it compatible with MaxText
```

Comment:
Please mention this is done in end_to_end/gemma/7b/1_test_gemma.sh since it is no longer there in this script

Reply:
Done